docs/src/main/asciidoc/spanner.adoc

```diff
@@ -1449,4 +1449,8 @@ spring.cloud.gcp.spanner.emulator-host=${EMULATOR_HOSTPORT}
 
 === Sample
 
-A https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/main/spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-sample[sample application] is available.
+Both the samples are listed below :
+
+1. https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/main/spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-template-sample[ Sample application for Spanner Template]
+
+2. https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/main/spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-repository-sample[ Sample application for Spanner Repository]
```

Comment:
```suggestion
2. https://github.com/GoogleCloudPlatform/spring-cloud-gcp/tree/main/spring-cloud-gcp-samples/spring-cloud-gcp-data-spanner-repository-sample[ Sample application using higher-level Spanner Repository capabilities]
```

Reply:
This is to make it clear when a user should be looking at one sample vs the other sample.